hagglehaul.Server/Models/GiveRating.cs

```diff
@@ -0,0 +1,7 @@
+namespace hagglehaul.Server.Models;
+
+public class GiveRating
+{
+    public string TargetUserEmail { get; set; }
+    public uint RatingGiven { get; set; }

```

Comment:
you should take in a tripId *instead* of `TargetUserEmail`. here's the reason why: we want to limit the number of reviews per trip to 1 (hence the `*hasBeenRated` booleans in each trip)

Reply:
Shouldn't be that expensive. one more database query can't hurt 🙃 

Ideally we'd want to implement it like this for security reasons.